static/css/restyle/restyle.less

```diff
@@ -1648,7 +1648,7 @@ h3.author .transfer-ownership {
 .category-landing .featured.three-col .addon.hovercard {
 
   &:hover {
-    height: 320px;
+    height: auto;
```

Comment:
This will break things elsewhere I believe, because that height is set so the margins will pull other bits of content up so the page content doesn't shift.